_layouts/default.html

```diff
@@ -1,173 +1,193 @@
-            <li><a href="{{ site.baseurl }}/pages/about.html">
-              About Us
-            </a></li>
-            <li><a href="{{ site.baseurl }}/pages/contribute.html">
-              How to contribute
-            </a></li>
-            <li><a href="https://github.com/auto-wcag/auto-wcag/">GitHub repository</a></li>
           </ul>
-        </li>
-
-      </ul>
-    </div><!--/.nav-collapse -->
-  </div>
-</nav>
-
-<main class="container">
-  {% if page_title %}
-    <h1>{{ page_title }}</h1>
-  {% endif %}
-
-  {{ content }}
-</main>
-
-<footer id="footer"><nav class="container"><div class="row">
-  <div class="col-sm-4">
-    <h2>Rule Creation</h2>
-    <ul>
-      <li><a href="{{ site.baseurl }}/pages/design/rule-design.html">
-        Rule design
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/design/review.html">
-        Review process
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/design/rule-template.html">
-        Rule template
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/design/style-guide.html">
-        Style guide
-      </a></li>
-    </ul>
-
-  </div> <div class="col-sm-4">
-    <h2>Rule Structure</h2>
-    <ul>
-      <li><a href="{{ site.baseurl }}/pages/structure/terminology.html">
-        Terminology
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/structure/test-modes.html">
-        Test modes
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/algorithms/">
-        Algorithms
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/structure/aggregation.html">
-        Result aggregation
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/structure/accessibility-support.html">
-        Accessibility support
-      </a></li>
-    </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+    
+    <!-- Begin page content -->
+    <main class="container">
+      {% if page_title %}
+        <div class="page-header">
+          <h1>{{ page_title }}</h1>
+        </div>
+      {% endif %}
+      <p>
+        {{ content }}
+      </p>
+    </main>
+    
+    <script src="{{ site.baseurl }}/assets/js/jquery.min.js"></script>
+    <script src="{{ site.baseurl }}/assets/js/bootstrap.min.js"></script>
+    <script>
+      $(document).ready(function () {
+        // append class names to table
+        $('table').each(function() {
+          $(this).addClass('table table-striped table-hover');
+        });
 
-  </div> <div class="col-sm-4">
-    <h2>About Auto-WCAG</h2>
-    <ul>
-      <li><a href="{{ site.baseurl }}/pages/about.html">
-        About Us
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/contribute.html">
-        How to contribute
-      </a></li>
-      <li><a href="https://www.w3.org/community/auto-wcag/about-us/">GitHub repository</a></li>
-    </ul>
-  </div>
+        // find every language-html and iframe next to eachother and flex them in 2 column layout
+        $('div.language-html')
```

Comment:
This too seems like it should work without JS.

Reply:
Same comment as above. Need to master Liquid Template generation.